src/clr-addons/history/history.service.ts

```diff
@@ -188,14 +188,14 @@ export class ClrHistoryService {
   }
 
   private encode(content: ClrHistoryModel[]): string {
-    const jsonString = btoa(JSON.stringify(content));
+    const jsonString = btoa(encodeURIComponent(JSON.stringify(content)));
```

Comment:
pls add a test to check this behaviour, so we don't run into it again in the future